test/ejabberd_sm_SUITE.erl

```diff
@@ -531,63 +581,6 @@ is_redis_running() ->
             false
     end.
 
-try_to_reproduce_race_condition(Config) ->
```

Comment:
delete_session could only be deleted by process from SID.
same with set_session. So, the testcase is not needed.

Though there is a case when a session is removed during netsplit detection, but it is a different testcase.